repo/packages/E/es-proxy/0/marathon.json

```diff
@@ -0,0 +1,33 @@
+{
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "{{ es-proxy.image }}:{{ es-proxy.version }}",
+      "network": "BRIDGE",
+      "portMappings": [
+        {
+          "containerPort": 5601,
+          "hostPort": 0,
+          "protocol": "tcp"
+        }
+      ]
+    }
+  },
+  "env": {
+    "ELASTICSEARCH_URL": "{{ es-proxy.es_url }}"
```

Comment:
No, by now it's not used, it's added is part of my work-in-progress on `haproxy-consul`. It **will** be used in nearest future.